server/routes/NodeIndexRoute.js

```diff
@@ -6,7 +6,35 @@
 const express = require('express');
 const nlpService = require("../services/NlpService");
 const dataService = require("../services/DataService");
-const router = express.Router();
+const crypto = require('crypto');
+var router = express.Router();
+var qs = require('querystring');
+var multer = require('multer');
+var path = require('path');
+var fs = require('fs');
+
+var DIR = "./server/data/";
+var storage = multer.diskStorage({
```

Comment:
Since we are using ES6 on backend also. you can take advantage of const, let and other similar features